casbin/persist/file_adapter/stringstream_adapter.cpp

```diff
@@ -0,0 +1,57 @@
+#include "pch.h"
+
+#include "./stringstream_adapter.h"
+#include "../../util/util.h"
+#include "../../exception/unsupported_operation_exception.h"
+
+namespace casbin {
+
+
+StringstreamAdapter :: StringstreamAdapter(std::stringstream && policy){
+    this->m_policy_stringstream = std::move(policy);
+}
+
+void StringstreamAdapter :: LoadPolicy(const std::shared_ptr<Model>& model) {
+    this->LoadPolicyStringStream(model, LoadPolicyLine);
+}
+
+void StringstreamAdapter :: LoadPolicyStringStream(const std::shared_ptr<Model>& model, 
+                                                   std::function<void(std::string, const std::shared_ptr<Model>&)> handler) { 
+    std::string line;
+    while(getline(this->m_policy_stringstream, line, '\n')){
```

Comment:
is `this` really necessary ?

Reply:
`this` is not necessary, in such way they write code in casbin

Reply:
all good then